pkg/local_object_storage/engine/control.go

```diff
@@ -54,3 +69,70 @@ func (e *StorageEngine) Close() error {
 
 	return nil
 }
+
+// executes op if execution is not blocked, otherwise returns blocking error.
+//
+// Can be called concurrently with setBlockExecErr.
+func (e *StorageEngine) exec(op func() error) error {
```

Comment:
I'll rename. But I didn't catch the relation to `onlyAlphabetEventHandler`.

Reply:
Both are wrappers with names that stress a condition when it is used. 

- `onlyAlpabetEventHandler` wraps event handler to activate in case if node is alphabet
- `execIfNothingBlocks` wraps closure to execute if no engine execution blockers are set.

Reply:
It's about the naming only or...?

Reply:
Naming only, yes.